sycl/test-e2e/MemorySanitizer/lit.local.cfg

```diff
@@ -0,0 +1,11 @@
+config.substitutions.append(
+    ("%device_msan_flags", "-Xarch_device -fsanitize=memory")
+)
+config.substitutions.append(
+    ("%force_device_msan_rt", "env UR_ENABLE_LAYERS=UR_LAYER_MSAN")
+)
+
+config.unsupported_features += ['cuda', 'hip']
+
+# https://github.com/intel/llvm/issues/16184
+config.unsupported_features += ['gpu-intel-gen12', 'gpu-intel-dg2']
```

Comment:
Assuming these fail consistently, I would prefer we made these `XFAIL` in the tests with an associated `XFAIL-TRACKER`. That would make it easier to track unintended restrictions.

Reply:
Makes sense.